.github/ISSUE_TEMPLATE/1.bug.yml

```diff
@@ -0,0 +1,55 @@
+name: Bug report 🐛
+description: Create a bug report for MUI Design kits.
+labels: ['status: needs triage']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please provide a searchable summary of the issue in the title above ⬆️.
+
+        Thanks for contributing by creating an issue! ❤️
+  - type: checkboxes
+    attributes:
+      label: Duplicates
+      description: Please [search the history](https://github.com/mui/mui-design-kits/issues) to see if an issue already exists for the same problem.
+      options:
+        - label: I have searched the existing issues
+          required: true
+  - type: checkboxes
+    attributes:
+      label: Latest version
+      description: We roll bug fixes and other improvements into new releases.
+      options:
+        - label: I have tested the latest version
+          required: true
+  - type: textarea
+    attributes:
+      label: Current behavior 😯
+      description: Describe what happens instead of the expected behavior.
+  - type: textarea
+    attributes:
+      label: Expected behavior 🤔
+      description: Describe what should happen.
+  - type: textarea
+    attributes:
+      label: Steps to reproduce 🕹
+      description: |
+        Please provide a link to a live example and an unambiguous set of steps to reproduce this bug.
+
+        Issues that we can't reproduce will be closed.
+      value: |
+        Link to live example:
+
+        Steps:
+        1.
+        2.
+        3.
+        4.
+  - type: textarea
+    attributes:
+      label: Context 🔦
```

Comment:
I wonder if we shouldn't put Context before expected behavior 🤔

Reply:
Interesting. Any objections to putting that first? Context seems to be the initial thing.

Reply:
Personally, I feel that the information is in the order of importance, the one I would use to scan a bug, first, I want to see what's the bug. But if you feel that context should be first, what do you think about proposing the change on the root issue template in https://github.com/mui/material-ui/blob/master/.github/ISSUE_TEMPLATE/1.bug.yml? What I like about the context is that I will create my own judgment on whether the expected vs. actual behavior is valid or invalid, **afterward**, I will check the context, see if this information change my worldview or reinforce my existing one.